src/components/DropdownTask.jsx

```diff
@@ -81,14 +95,41 @@ const DropdownTask = ({ columnId, taskId, handleEdit }) => {
                     </ListItemIcon>
                     <ListItemText primary="Edit" />
                 </MenuItem>
-                <MenuItem onClick={handleDelete}>
+                <MenuItem onClick={openSnackbar}>
                     <ListItemIcon>
                         <Delete fontSize="default" />
                     </ListItemIcon>
                     <ListItemText primary="Remove" />
                 </MenuItem>
+                <MenuItem>
+                    <ListItemIcon>
+                        <Archive fontSize="default" />
+                    </ListItemIcon>
+                    <ListItemText primary="Archive" />
+                </MenuItem>
             </Menu>
+            <Snackbar
+                classes={{ root: classes.snackbar }}
+                open={open}
+                anchorOrigin={{ horizontal: 'right', vertical: 'top' }}
```

Comment:
Näissä pitäis mun mielestä olla joku tausta kun nyt se alla oleva kolumni näkyy messagen takana kun on läpinäkyvä:

![image](https://user-images.githubusercontent.com/42170182/94112253-d6543080-fe4d-11ea-8416-1464afbd3096.png)